src/components/Disclosure/Disclosure.scss

```diff
@@ -6,59 +6,50 @@ $trigger: #{$block}__trigger;
 $gap: 8px;
 
 #{$block} {
-    #{$trigger} {
+    &_enter_active {
+        animation-name: #{variables.$ns}disclosure-expanded;
+        animation-duration: 0.5s;
+    }
+    &_exit_active {
+        display: block;
+        animation-name: #{variables.$ns}disclosure-collapsed;
+        animation-duration: 0.1s;
```

Comment:
Current animation looks like a FOUT (Flash Of Unstyled Text). IMO better without it completely

Reply:
Discussed with Yuriy and made some changes. Could you please take a look at the last storybook?